src/_data/site.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function() {
+  return {
+	title: "swup",
+	description: "Complete, flexible, extensible, and easy-to-use page transition library for your server-side rendered website.",
+    url: process.env.SITE_URL || "development",
```

Comment:
Should this say `swup.js.org` instead of `development`?

Reply:
OH! Yes of course. Also, something just broke. I'll look into it.

Reply:
...working now.